test/DepthImageToLaserScanTest.cpp

```diff
@@ -36,6 +36,8 @@
 // Bring in gtest
 #include <gtest/gtest.h>
 
+#include <sensor_msgs/image_encodings.hpp>
```

Comment:
Ah, right.  To be honest, I still don't quite understand the order of C vs C++ includes that we prefer.  I didn't see it in the style guide.  Should I move this to the top, then?